### PR TITLE
[DOCS-7526] adding support for US5 and EU - updating banners

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
[DOCS-7526](https://datadoghq.atlassian.net/browse/DOCS-7526)

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

[DOCS-7526]: https://datadoghq.atlassian.net/browse/DOCS-7526?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/mobile_app_testing/_index.md [hunk 1]
```diff
@@ -23,14 +22,19 @@ cascade:
     tags: ['mobile_testing']
 ---
 
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites. Support for US3 and AP1 coming soon.</div>
```
Comment (rtrieu) [inline content/en/mobile_app_testing/_index.md:26]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is Generally Available for US1, US5, and EU sites.</div>
```

Comment (rtrieu) [reply content/en/mobile_app_testing/_index.md:26]: Suggestion to match existing styling of similar text, and removing promises as a matter of docs policy.

File: content/en/mobile_app_testing/_index.md [hunk 2]
```diff
@@ -23,14 +22,19 @@ cascade:
...
 {{< /site-region >}}
 
-{{< site-region region="us" >}}
-<div class="alert alert-info">Mobile Application Testing is in limited availability and is only supported for the Datadog US1 site.</div>
+{{< site-region region="us3,ap1" >}}
+<div class="alert alert-warning">Support for Mobile Application Testing is coming soon.</div>
```
Comment (rtrieu) [inline content/en/mobile_app_testing/_index.md:30]: We only document the product as it is now, without making promises for the future. My suggestion would be to leave this as is until support for US3 and AP1 is available.

File: content/en/mobile_app_testing/mobile_app_tests/_index.md [hunk 3]
```diff
@@ -16,12 +16,16 @@ further_reading:
...
 ---
 
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites.</div>
+{{< /site-region >}}
+
+{{< site-region region="us3,ap1" >}}
+<div class="alert alert-warning">Support for Mobile Application Testing is coming soon.</div>
```
Comment (rtrieu) [inline content/en/mobile_app_testing/mobile_app_tests/_index.md:24]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

File: content/en/mobile_app_testing/_index.md (hunk 2 above)
Comment (rtrieu) [inline content/en/mobile_app_testing/_index.md:30]: OR:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

File: content/en/mobile_app_testing/settings/_index.md [hunk 4]
```diff
@@ -14,13 +14,18 @@ further_reading:
...
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites.</div>
 {{< /site-region >}}
 
-{{< site-region region="us" >}}
-<div class="alert alert-info">Mobile Application Testing is in limited availability and is only supported for the Datadog US1 site.</div>
+{{< site-region region="us3,ap1" >}}
+<div class="alert alert-warning">Support for Mobile Application Testing is coming soon.</div>
```
Comment (rtrieu) [inline content/en/mobile_app_testing/settings/_index.md:22]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

OR leave as is until support is available for these regions

**Review by rtrieu — CHANGES_REQUESTED** [review]
Left some feedback. Let me know if you have any questions!

File: content/en/mobile_app_testing/mobile_app_tests/_index.md [hunk 5]
```diff
@@ -16,12 +16,16 @@ further_reading:
   text: "Best practices for maintaining end-to-end tests"
 ---
 
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites.</div>
```
Comment (rtrieu) [inline content/en/mobile_app_testing/mobile_app_tests/_index.md:20]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is Generally Available for US1, US5, and EU sites.</div>
```

File: content/en/mobile_app_testing/mobile_app_tests/results.md [hunk 6]
```diff
@@ -13,12 +13,16 @@ further_reading:
   text: "Learn about the Events Explorer"
 ---
 
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites.</div>
```
Comment (rtrieu) [inline content/en/mobile_app_testing/mobile_app_tests/results.md:17]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is Generally Available for US1, US5, and EU sites.</div>
```

File: content/en/mobile_app_testing/mobile_app_tests/steps.md [hunk 7]
```diff
@@ -13,12 +13,16 @@ further_reading:
   text: "Learn about advanced options in mobile tests"
 ---
 
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites.</div>
```
Comment (rtrieu) [inline content/en/mobile_app_testing/mobile_app_tests/steps.md:17]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is Generally Available for US1, US5, and EU sites.</div>
```

**Review by rtrieu — COMMENTED** [review]
I had missed a few.

**Comment by aliciascott** [comment]
> I had missed a few.

I just updated a few others we missed too if you can do a re-review :)

**Review by rtrieu — CHANGES_REQUESTED** [review]
Looks good!

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**